powersimdata/design/investment/README.md

```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
-- Map ["Technology","r" (region)] to ReEDS regional capital cost multipliers. Keep (arbitrary) subclasses for renewables.
+- If a technology is wind or wind_offshore or csp, regions are in rs (wind resource region), so keep rs is kept as the region to map. If technology is another tech, regions are in rb (BA region), so keep rb as the region to map.
+- Map ["Technology", "r" (region)] to ReEDS regional capital cost multipliers. Keep (arbitrary) subclasses for renewables.
+- Apply inflation to scale 2018 dollars to present.
 - Final calculations.
 
-**Warning**: there's a lot of messy code here right now because I'm manually dropping a lot of subtypes of technologies so we only have one type.
-
-## Mapping functions
-
-**sjoin_nearest**: joins a geodataframe of Points and Polygons/Multipolygons. Used in `points_to_polys`.
 
-**points_to_polys**: joins a dataframe (which includes lat and lon columns) with a shapefile. Used in `bus_to_neem_reg` and `plant_to_reeds_reg`.
+### Mapping functions
 
+`sjoin_nearest`: joins a geodataframe of Points and Polygons/Multipolygons. Used in `points_to_polys`.
 
+`points_to_polys`: joins a dataframe (which includes lat and lon columns) with a shapefile. Used in `bus_to_neem_reg` and `plant_to_reeds_reg`.
 
-**Functions used for AC regional multiplier mapping**
+#### Functions used for AC regional multiplier mapping
 
-**bus_to_neem_reg**: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
+`bus_to_neem_reg`: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
 
-**write_bus_neem_map**: maps all base_grid bus locations to NEEM regions and produces a csv mapping file: regionsNEEM.shp. This csv is used in `_calculate_ac_inv_costs`.
+`write_bus_neem_map`: maps all base_grid bus locations to NEEM regions and produces a csv mapping file: regionsNEEM.shp. This csv is used in `_calculate_ac_inv_costs`.
 
 
+#### Functions used for generation/storage regional multiplier mapping
 
-**Functions used for generation/storage regional multiplier mapping**
+`write_poly_shapefile`: Using a csv with specified geometry, creates the shapefile for ReEDS wind resource regions: rs.shp. This shp is used in `plant_to_reeds_reg`.
 
-**write_poly_shapefile**: Using a csv with specified geometry, creates the shapefile for ReEDS wind resource regions: rs.shp. This shp is used in `plant_to_reeds_reg`.
+`plant_to_reeds_reg`: maps plant locations to ReEDS regions. Used in `_calculate_gen_inv_costs`.
 
-**plant_to_reeds_reg**: maps plant locations to ReEDS regions. Used in `_calculate_gen_inv_costs`.
 
+### Sources
 
-### Sources (more specifics in data folder README.md)
+See **ATTRIBUTION.md**.
```

Comment:
you could create a link to the file here

Reply:
Done.